aws/table_aws_cloudwatch_metric_statistic_data_point.go

```diff
@@ -0,0 +1,173 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+	cloudwatchv1 "github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudWatchMetricStatisticDataPoint(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudwatch_metric_statistic_data_point",
+		Description: "AWS CloudWatch Metric Statistic Data Point",
+		List: &plugin.ListConfig{
+			Hydrate: listCloudWatchMetricStatisticDataPoints,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValue"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "metric_name",
+					Require: plugin.Required,
+				},
+				{
+					Name:    "namespace",
+					Require: plugin.Required,
+				},
+				{
+					Name:      "timestamp",
+					Operators: []string{">", ">=", "=", "<", "<="},
+					Require:   plugin.Required,
+				},
+				{
+					Name:    "period",
```

Comment:
On any of the optional key quals, I think we need to add `CacheMatch: Exact`. For instance, if caching is enabled, if a user runs a query without any optional quals, and then runs a query with one or more optional quals specified,  they'll get unexpected results since the API should return 2 different data sets.

Reply:
updated.